app/controllers/application_controller.rb

```diff
@@ -1,2 +1,4 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  include SessionsHelper
```

Comment:
If you have SessionsHelper included in Application Controller it probably means that you probably have code in SessionsHelper that is meant for Controller.